docs/sources/mimir/references/architecture/deployment-modes/index.md

```diff
@@ -41,46 +53,26 @@ Monolithic mode can be horizontally scaled out by deploying multiple Grafana Mim
 Because monolithic mode requires scaling all Grafana Mimir components together, this deployment mode isn't recommended for large-scale deployments.
 {{< /admonition >}}
 
-## Microservices mode
+## About microservices mode
 
-In microservices mode, components are deployed in distinct processes. Scaling is per component, which allows for greater flexibility in scaling and more granular failure domains. Microservices mode is the preferred method for a production deployment, but it is also the most complex.
+Microservices mode deploys each component in separate processes, enabling independent scaling and creating granular failure domains. This deployment mode is recommended for production environments that require high availability and flexible scaling.
```

Comment:
I'd rather not qualify when microservices mode is recommended for production, it's always recommended. We'd only expect people to _not_ run it in production in special circumstances that tend to be very unique to their environment or organization.